lib/init.g

```diff
@@ -406,35 +406,35 @@ CallAndInstallPostRestore( function()
   MakeImmutable( GAPInfo.TermEncoding );
 end );
 
-
-
-BindGlobal( "ShowKernelInformation", function()
-  local sysdate, linelen, indent, btop, vert, bbot, print_info,
-        config, str, gap;
+BindGlobal("PrintInfo",
```

Comment:
That's a pretty generic name, maybe not the best to use for an internal helper? Perhaps `_PrintInfo` or s/o?